Patches.py

```diff
@@ -1511,8 +1567,56 @@ def set_entrance_updates(entrances):
     if 'ganondorf' in world.settings.misc_hints:
         buildGanonText(world, messages)
 
+
+    logger = logging.getLogger('')
+    #Patch actor overrides for freestanding items. Need to figure out how to fix this for MQ
```

Comment:
What do you mean "Fix this for MQ"?

Reply:
Nothing anymore. That comment was just from very early on during the development of this setting when I didn't have MQ freestanding items shuffled.